src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -622,6 +640,119 @@ namespace webnn_native { namespace ie {
         return {};
     }
 
+    MaybeError Graph::AddGru(const op::Gru* gru) {
```

Comment:
Need you consider the case for options->layout = rzn?

Need you check for `direction: a MLRecurrentNetworkDirection. The processing direction of the input sequence. When set to "both", the size of the first dimension of the weight and the bias tensor shapes must be 2, and the input is processed in both directions.` ?

Reply:
Added the check code.